src/containers/NoteForm/NoteForm.js

```diff
@@ -11,15 +11,20 @@ export class NoteForm extends Component {
       items: []
     }
   }
+  
+  componentDidMount = () => {
+    this.setState({ items: [{ id: Date.now(), description: '', noteID: this.state.note.id }] })
+  }
 
   //  this.props.items || 
 
   handleOnChange = (e) => {
     const { name, value } = e.target
+    const { items, note } = this.state
```

Comment:
I like the destructuring here. Definitely improves the readability on subsequent lines.

Reply:
👍